cwms-radar-client/src/test/resources/radar/v2/json/binarytimeseries.json

```diff
@@ -26,7 +26,7 @@
           108,
           100
       ],
-      "value-url": "HelloWorld.com",
+      "value-url": "https://localhost:8888/timeseries/binary/TEST.Binary.Inst.1Hour.0.MockTest/value",
```

Comment:
The URL should have query parameters, I should have linked the UML diagram with the example:


![](https://github.com/USACE/cwms-data-api/blob/develop/docs/uml/bin_largedata_ts_value_url.png)

`https://localhost:59947/cwms-data/timeseries/binary/Test.Flow.Inst.1Hour.0.raw/value?office=SPK&version-date&date=2008-05-01T15%3A00%3A00Z`